trunk/SUAVE/Methods/Geometry/Two_Dimensional/Planform/__init__.py

```diff
@@ -8,4 +8,4 @@
 from horizontal_tail_planform_raymer import horizontal_tail_planform_raymer
 from vertical_tail_planform_raymer import vertical_tail_planform_raymer
 from wing_fuel_volume import wing_fuel_volume
-from populate_control_sections import populate_control_sections
+#from populate_control_sections import populate_control_sections
```

Comment:
Why is this commented out?

Reply:
I was trying to debug the control surface scripts. I will undo the comment.